en/sensor/sf1xx_lidar_setup.md

```diff
@@ -26,3 +27,18 @@ Use the `SENS_EN_SF1XX` parameter to select the lidar model and then reboot.
 * `2` SF10/b
 * `3` SF10/c
 * `4` SF11/c
+
+For PX4v2 you also need to add it to the make file:
```

Comment:
Please change the first bit to "For older targets where you need to flash `px4fmu-v2_default`,... "